hack/release/common.sh

```diff
@@ -66,136 +53,165 @@ authenticatePrivateRepo() {
+  find "website/content/en/v${short_version}/" -type f -print0 | xargs perl -i -p -e "s/{{< githubRelRef >}}/\/v${short_version}\//g;"
 
-    rm -rf website/content/en/docs
-    mkdir -p website/content/en/docs
-    cp -r website/content/en/${RELEASE_MINOR_VERSION}/* website/content/en/docs/
+  rm -rf website/content/en/docs
+  mkdir -p website/content/en/docs
+  cp -r "website/content/en/v${short_version}/"* website/content/en/docs/
 }
 
 removeOldWebsiteDirectories() {
-  local n=3
+  local n=3 last_n_versions all
+
   # Get all the directories except the last n directories sorted from earliest to latest version
   # preview, docs, and v0.32 are special directories that we always propagate into the set of directory options
   # Keep the v0.32 version around while we are supporting v1beta1 migration
   # Drop it once we no longer want to maintain the v0.32 version in the docs
-  last_n_versions=$(find website/content/en/* -maxdepth 0 -type d -name "*" | grep -v "preview\|docs\|v0.32" | sort | tail -n "$n")
+  last_n_versions=$(find website/content/en/* -maxdepth 0 -type d -name "*" | grep -v "preview\|docs\|v0.32" | sort | tail -n "${n}")
   last_n_versions+=$(echo -e "\nwebsite/content/en/preview")
   last_n_versions+=$(echo -e "\nwebsite/content/en/docs")
   last_n_versions+=$(echo -e "\nwebsite/content/en/v0.32")
   all=$(find website/content/en/* -maxdepth 0 -type d -name "*")
+
   ## symmetric difference
-  comm -3 <(sort <<< $last_n_versions) <(sort <<< $all) | tr -d '\t' | xargs -r -n 1 rm -r
+  # shellcheck disable=SC2086
+  comm -3 <(sort <<< ${last_n_versions}) <(sort <<< ${all}) | tr -d '\t' | xargs -r -n 1 rm -r
 }
 
 editWebsiteConfig() {
-  RELEASE_VERSION=$1
-  yq -i ".params.latest_release_version = \"${RELEASE_VERSION}\"" website/hugo.yaml
+  local version="${1}"
+
+  yq -i ".params.latest_release_version = \"v${version}\"" website/hugo.yaml
 }
 
 # editWebsiteVersionsMenu sets relevant releases in the version dropdown menu of the website
 # without increasing the size of the set.
 # It uses the current version directories (ignoring the docs directory) to generate this list
 editWebsiteVersionsMenu() {
-  VERSIONS=($(find website/content/en/* -maxdepth 0 -type d -name "*" | xargs -r -n 1 basename | grep -v "docs\|preview"))
-  VERSIONS+=('preview')
+  local versions version
+
+  # shellcheck disable=SC2207
+  versions=($(find website/content/en/* -maxdepth 0 -type d -name "*" -print0 | xargs -r -n 1 basename | grep -v "docs\|preview"))
+  versions+=('preview')
 
   yq -i '.params.versions = []' website/hugo.yaml
 
-  for VERSION in "${VERSIONS[@]}"; do
-    yq -i ".params.versions += \"${VERSION}\"" website/hugo.yaml
+  for version in "${versions[@]}"; do
+    yq -i ".params.versions += \"${version}\"" website/hugo.yaml
   done
 }
+
+# updateKarpenterCoreGoMod bumps the karpenter-core go.mod to the release version so that the
+# karpenter and karpenter-core release versions match
+updateKarpenterCoreGoMod(){
```

Comment:
I think we can drop this function from the stable PR. We ensure we pin the version through a helm update ahead-of-time so there's no need for this function to exist right now AFAIK

Reply:
Will do.